### PR TITLE
Aftershock: Blood bionics

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Mods "Aftershock: New combat bionics"

#### Purpose of change
More chrome.

#### Describe the solution
The new bionics are:

- Blood neurology: A powerful stat enhancer that also diminishes pain and its effects on intelligence. Prevents you from dying from head trauma although you still suffer when your main brain gets shaken.
- Synthetic Marrow: A cardio increase, grants you a slightly weakened  extra dex, strength and speed from the synthblood effect.

#### Testing
Used both bionics a bit.

### CODE REVIEW

File: data/mods/Aftershock/effects.json [hunk 1]
```diff
@@ -178,6 +178,19 @@
     "blood_analysis_description": "Synthetic blood",
     "show_in_info": true
   },
+  {
+    "type": "effect_type",
+    "id": "synth_blood_permanent",
+    "name": [ "Synthblood" ],
+    "desc": [
+      "Your breaths instinctively grows more shallow as the synthetic blood carries oxygen throughout your body, increasing your speed, stamina, and general endurance."
```
Comment (John-Candlebury) [inline data/mods/Aftershock/effects.json:186]:
```suggestion
      "Your breaths instinctively grow shallow as the synthetic blood carries oxygen throughout your body, increasing your speed, stamina, and general endurance."
```

File: data/mods/Aftershock/player/bionics.json [hunk 2]
```diff
@@ -83,6 +83,38 @@
...
+    "id": "afs_bio_blood_plus",
+    "type": "bionic",
+    "name": { "str": "Synthetic Marrow" },
+    "description": "The interior of your femurs have been scrapped clean, the marrow replaced with a Mercurial brand bio-factory that produces high efficiency blood analogues.  The blood substitutes are fine-tuned for combat, increasing stamina, disease resistance and clotting speed.",
+    "cant_remove_reason": "The biofactory has perfectly integrated with your body, removing it would be pointless and potentially fatal.",
+    "occupied_bodyparts": [ [ "leg_l", 2 ], [ "leg_r", 2 ] ],
+    "enchantments": [
+      {
+        "ench_effects": [ { "effect": "synth_blood_permanent", "intensity": 2 } ],
+        "values": [ { "value": "CARDIO_MULTIPLIER", "multiply": 1.2 } ]
```
Comment (GuardianDll) [inline data/mods/Aftershock/player/bionics.json:96]: Multiplier 1.2 means 2.2x increase , that's how enchantment math works 
```suggestion
        "values": [ { "value": "CARDIO_MULTIPLIER", "multiply": 0.2 } ]
```

File: data/mods/Aftershock/player/bionics.json [hunk 3]
```diff
@@ -83,6 +83,38 @@
...
+    "description": "Your blood is laced with neuron-analogue nanites, allowing your entire circulatory system to function as a secondary brain.  This radical modification increases your intelligence, reflexes and perception by 4, increases your pain tolerance and allows you to survive brain damage that would otherwise prove fatal.",
+    "occupied_bodyparts": [ [ "leg_l", 2 ], [ "leg_r", 2 ] ],
+    "enchantments": [
+      {
+        "values": [
+          { "value": "INTELLIGENCE", "add": 4 },
+          { "value": "PERCEPTION", "add": 4 },
+          { "value": "DEXTERITY", "add": 4 },
+          { "value": "PAIN", "multiply": -0.33 },
+          { "value": "PAIN_PENALTY_MOD_INT", "add": 4 }
```
Comment (GuardianDll) [inline data/mods/Aftershock/player/bionics.json:113]: I assume you try to decrease stat loss from pain, not increase 
```suggestion
          { "value": "PAIN_PENALTY_MOD_INT", "add": - 4 }
```

File: data/mods/Aftershock/player/bionics.json (hunk 2 above)
Comment (John-Candlebury) [reply data/mods/Aftershock/player/bionics.json:96]: Oh no this is chromed up warmachine-psycho tier stuff, its meant to be a duplication of stamina.

File: data/mods/Aftershock/player/bionics.json [hunk 4]
```diff
@@ -83,6 +83,38 @@
...
+    "description": "Your blood is laced with neuron-analogue nanites, allowing your entire circulatory system to function as a secondary brain.  This radical modification increases your intelligence, reflexes and perception by 4, increases your pain tolerance and allows you to survive brain damage that would otherwise prove fatal.",
+    "occupied_bodyparts": [ [ "leg_l", 2 ], [ "leg_r", 2 ] ],
+    "enchantments": [
+      {
+        "values": [
+          { "value": "INTELLIGENCE", "add": 4 },
+          { "value": "PERCEPTION", "add": 4 },
+          { "value": "DEXTERITY", "add": 4 },
+          { "value": "PAIN", "multiply": -0.33 },
+          { "value": "PAIN_PENALTY_MOD_INT", "add": - 4 }
```
Comment (John-Candlebury) [inline data/mods/Aftershock/player/bionics.json:113]:
```suggestion
          { "value": "PAIN_PENALTY_MOD_INT", "add": -4 }
```

File: data/mods/Aftershock/items/cbms.json [hunk 5]
```diff
@@ -37,6 +37,24 @@
     "price": "15 kUSD",
     "difficulty": 5
   },
+  {
+    "id": "afs_bio_blood_plus",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": { "str": "Synthblood Biofactory CBM" },
+    "description": "A mercurial brand biofactory that replaces the user's blood with a synthetic substitute.  Primarily desinged to boost combat performance, its common among the corporation's own security forces.  The biofactory can't be removed once integrated witha a host.",
```
Comment (casswedson) [inline data/mods/Aftershock/items/cbms.json:45]:
```suggestion
    "description": "A mercurial brand biofactory that replaces the user's blood with a synthetic substitute.  Primarily desinged to boost combat performance, its common among the corporation's own security forces.  The biofactory can't be removed once integrated with a host.",
```

File: data/mods/Aftershock/items/cbms.json [hunk 6]
```diff
@@ -37,6 +37,24 @@
     "price": "15 kUSD",
     "difficulty": 5
   },
+  {
+    "id": "afs_bio_blood_plus",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": { "str": "Synthblood Biofactory CBM" },
+    "description": "A mercurial brand biofactory that replaces the user's blood with a synthetic substitute.  Primarily desinged to boost combat performance, its common among the corporation's own security forces.  The biofactory can't be removed once integrated with a host.",
```
Comment (Maleclypse) [inline data/mods/Aftershock/items/cbms.json:45]:
```suggestion
    "description": "A Mercurial Genomics brand biofactory that replaces the user's blood with a synthetic substitute.  Primarily designed to boost combat performance, its common among the corporation's own security forces.  The biofactory can't be removed once integrated with a host.",
```

**Review by Maleclypse — APPROVED** [review]

**Comment by Maleclypse** [comment]
I love these. Thanks!

**Pull request merged**